Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/AlertMessage.java

```diff
@@ -71,6 +75,17 @@ public Dialog onCreateDialog(Bundle savedInstanceState)
                 NativeLibrary.NotifyAlertMessageLock();
               });
     }
+
+    if (isWarning)
+    {
+      builder.setNeutralButton("IGNORE FOR THIS SESSION", (dialog, which) ->
```

Comment:
This string should be in a resources file and should not be in allcaps.

Reply:
The string is converted to all caps when displayed but I now use a string resource that matches Qt.